app/controllers/surveys_controller.rb

```diff
@@ -13,6 +13,7 @@ def send_survey
       return render json: {error: :survey_not_found}, status: :not_found
     end
 
+    @survey.generate_models
```

Comment:
What kind of check would suffice here? Any existing questions models associated with this survey means don't regenerate?